conf/upgrade/upgrade_ui.yaml

```diff
@@ -0,0 +1,3 @@
+---
```

Comment:
@dahorak @petr-balogh 
will we need modifications is Jenkins?

Reply:
It is possible to supply this config file as `CLUSTER_CONF` parameter (in that case, no modification is required) ... but if we would like to have it slightly more smooth (like for UI deployment - via `UI_DEPLOY` checkbox) than yes, we will have to update the deployment and other jobs.

Reply:
Created [issue#77](https://gitlab.cee.redhat.com/ocs/ocs4-jenkins/-/issues/77) in our ocs4-jenkins repo for this.

Reply:
Related to the `UI_UPGRADE` parameter for jenkins job - if this configuration file is used, should be together also set `UPGRADE["upgrade"] = True` parameter (used for normal upgrade)?
And in the Jenkins job - should user select both parameters `UPGRADE` and `UI_UPGRADE`  together (for UI upgrade) or only the second one?

Reply:
@dahorak it sounds redundant but is there any other option?

Reply:
I've discussed it with Petr and we decided, that both parameters (`UPGRADE` and `UI_UPGRADE`) should be selected together and I've implemented it that way in [MR!596](https://gitlab.cee.redhat.com/ocs/ocs4-jenkins/-/merge_requests/596) (with check which will fail the job with error message, if `UI_UPGRADE` is selected without `UPGRADE` parameters).